### PR TITLE
user/LÖVE: new package

### PR DESCRIPTION
**Pull request by sudanchapagain**
## Description

new package.

Describe what your pull request does here. For new packages, this is the
purpose of the package, for other changes this is what your change does,
for trivial package updates you may remove this.

## Checklist

Before this pull request is reviewed, certain conditions must be met.

The following must be true for all changes:

- [x] I have read [CONTRIBUTING.md](https://github.com/chimera-linux/cports/blob/master/CONTRIBUTING.md)

The following must be true for template/package changes:

- [x] I have read [Packaging.md](https://github.com/chimera-linux/cports/blob/master/Packaging.md#correct_templates)
- [x] I have built and tested my changes on my machine

### CODE REVIEW

**Comment by JamiKettunen** [comment]
`user/love` in commit message please since that's what the package itself is called

**Comment by sudanchapagain** [comment]
@JamiKettunen  done :)

File: user/love/template.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "sdl-devel",
+    "zlib-ng-devel",
+]
+pkgdesc = "Lua 2D Graphics Library"
+maintainer = "sudan <sudanchapagain@proton.me>"
+license = "Zlib"
+url = "https://love2d.org"
+source = f"https://github.com/love2d/love/releases/download/{pkgver}/love-{pkgver}-linux-src.tar.gz"
+sha256 = "066e0843f71aa9fd28b8eaf27d41abb74bfaef7556153ac2e3cf08eafc874c39"
+
```
Comment (q66) [inline user/love/template.py:25]: spurious empty line

File: user/love/template.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+pkgname = "love"
+pkgver = "11.5"
+pkgrel = 0
+build_style = "gnu_configure"
+make_dir = "."
```
Comment (q66) [inline user/love/template.py:5]: this needs a comment why

File: user/love/template.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+hostmakedepends = ["automake", "libtool", "pkgconf"]
```
Comment (q66) [inline user/love/template.py:6]: should probably be slibtool

File: user/love/template.py (hunk 2 above)
Comment (sudanchapagain) [reply user/love/template.py]: so, i looked at Alpine's APKBUILD file and tried to build it and it's from there. I built and tested it again without it and it works. sorry.

File: user/love/template.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    "libtheora-devel",
+    "libvorbis-devel",
+    "libx11-devel",
+    "luajit-devel",
+    "mpg123-devel",
+    "openal-soft-devel",
+    "sdl-devel",
+    "zlib-ng-devel",
+]
+pkgdesc = "Lua 2D Graphics Library"
```
Comment (q66) [inline user/love/template.py:18]: no titlecase, i.e. `Lua 2D graphics library`

Comment (sudanchapagain) [reply user/love/template.py]: done

**Pull request merged**